docs/ELASTIC_9_UPGRADE_PLAN.md

```
@@ -0,0 +1,181 @@
+
+#### 3. Upgrade Kibana
+*   **Container Switchover**: Stop the active Kibana service, update its container compose or Quadlet definition to reference the matching `9.5.x` Wolfi Kibana image, and launch:
+    ```bash
+    systemctl --user stop kib01.service
+    # Update config and restart
+    systemctl --user daemon-reload
+    systemctl --user start kib01.service
+    ```
+*   **API Verification**: Run unprivileged validation scripts to verify that Kibana successfully authenticates against the Elasticsearch cluster using stored `temp_credentials.txt` or vault secrets.
+
+#### 4. Upgrade Elastic Fleet Integration + Elastic Agent Related
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)